src/extraction.js

```diff
@@ -88,7 +89,8 @@ module.exports = {
 
     const bucketName = process.env.GCP_EXPORT_BUCKET;
 
-    const filename = `${extractionType}.ndjson.gz`;
+    // All extract jobs use multi-file wildcard output
+    const filename = `${extractionType}-*.ndjson.gz`;
```

Comment:
So now, we're going to produce 10+ files for _every_ extraction job?  Even podcast/episode metadata?

Reply:
I guess I was hoping we could just catch the "file too big" error at runtime, and then retry once with the wildcard'd filename.  But maybe that's a significant code complexity to handle both copy formats.

Could also just listObjects the resulting subfolder, and copy everything in there?  That also solves the filename brittleness.

Reply:
Ah, interesting. So the number of files is either table size, or related to partitioned tables.